app/serializers/api/public/v1/serializable_location.rb

```diff
@@ -0,0 +1,36 @@
+module API
+  module Public
+    module V1
+      class SerializableLocation < JSONAPI::Serializable::Resource
+        type "locations"
+
+        attributes :code,
+                   :latitude,
+                   :location_name,
+                   :longitude,
+                   :postcode,
+                   :region_code
+
+        attribute :street_address_1 do
+          @object.address1
+        end
+
+        attribute :street_address_2 do
+          @object.address2
+        end
+
+        attribute :city do
+          @object.address3
+        end
+
+        attribute :county do
+          @object.address4
+        end
+
+        attribute :recruitment_cycle_year do
```

Comment:
@misaka can we remove this? We're already scoping to the recruitment cycle year so this duplicate information

Reply:
I thought it can be a nice-to-have to expose this, kind of like how we expose it, and provider code, in the course. Seems to be a nice to have because you might, say, in the clien have the URL of the location of this resource, but maybe not all the components of it ... and inspecting the URL could be annoying.

Reply:
Can I resolve this conversation?

Reply:
Just checked, and there's a good reason not to include this ... it forces the query to be N+1, or to include the `recruitment_cycle` into it. Queries should be fast by default, so we should doing either of these ... let the client include the `recruitment_cycle` and pull out the year from it if they need it. ✔️

Reply:
in other words remove it

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1513/commits/842f178f0b54df1190fc13a9f28a23939e855459